src/xrpld/app/tx/detail/NFTokenUtils.cpp

```diff
@@ -1081,5 +1092,115 @@ tokenOfferCreateApply(
     return tesSUCCESS;
 }
 
+TER
+checkTrustlineAuthorized(
+    ReadView const& view,
+    AccountID const id,
+    beast::Journal const j,
+    Issue const& issue)
+{
+    // Only valid for custom currencies
+    XRPL_ASSERT(
+        !isXRP(issue.currency),
+        "ripple::nft::checkTrustlineAuthorized : valid to check.");
+
+    if (view.rules().enabled(fixEnforceNFTokenTrustlineV2))
+    {
+        auto const issuerAccount = view.read(keylet::account(issue.account));
+        if (!issuerAccount)
+        {
+            JLOG(j.debug()) << "ripple::nft::checkTrustlineAuthorized: can't "
+                               "receive IOUs from non-existent issuer: "
+                            << to_string(issue.account);
+
+            return tecNO_ISSUER;
```

Comment:
If it's not reachable, it may be worth adding `// LCOV_EXCL_LINE` so that we don't get codecov warnings